package.json

```diff
@@ -17,12 +17,12 @@
     "build": "gulp build-release-packages",
     "bazel:buildifier": "find . -type f \\( -name \"*.bzl\" -or -name WORKSPACE -or -name BUILD -or -name BUILD.bazel \\) ! -path \"*/node_modules/*\" | xargs buildifier -v --warnings=attr-cfg,attr-license,attr-non-empty,attr-output-default,attr-single-file,constant-glob,ctx-args,depset-iteration,depset-union,dict-concatenation,duplicated-name,filetype,git-repository,http-archive,integer-division,load,load-on-top,native-build,native-package,output-group,package-name,package-on-top,redefined-variable,repository-name,same-origin-load,string-iteration,unused-variable",
     "bazel:format-lint": "yarn -s bazel:buildifier --lint=warn --mode=check",
-    "dev-app": "gulp serve:devapp",
+    "dev-app": "bazel run //src/dev-app:devserver",
     "test": "bazel test //src/... --test_tag_filters=-e2e,-browser:firefox-local --build_tag_filters=-browser:firefox-local --build_tests_only",
     "test-firefox": "bazel test //src/... --test_tag_filters=-e2e,-browser:chromium-local --build_tag_filters=-browser:chromium-local --build_tests_only",
     "lint": "gulp lint && yarn -s bazel:format-lint",
     "e2e": "bazel test //src/... --test_tag_filters=e2e",
-    "deploy": "gulp deploy:devapp",
+    "deploy": "echo 'Not supported yet. Tracked with COMP-230'",
```

Comment:
I moved this to the queue since I want to keep this deploy functional; we'll likely want to do some amount of IE and mobile testing with the MDC components as we progress